File: Luna.Models.Tasks.View/CardAttributes/StatusView.cs
using Luna.Models.Tasks.Domain.CardAttributes;

namespace Luna.Models.Tasks.View.CardAttributes;

public class StatusView
{
	public Guid Id { get; set; }

	public String Name { get; set; }

	public String Color { get; set; }

	public StatusView(StatusDomain statusDomain)
	{
		Id = statusDomain.Id;
		Name = statusDomain.Name;
		Color = statusDomain.Color;
	}
}

File: luna/src/models/card/statusView.ts
import {Guid} from "guid-typescript";

export default class StatusView {
    constructor(id: Guid, name: string, color: string) {
        this.id = id;
        this.name = name;
        this.color = color;
    }

    public id: Guid;
    public name: string;
    public color: string;
}